src/_includes/docs/swift-package-manager/how-to-enable-disable.md

```diff
@@ -0,0 +1,78 @@
+## How to turn on Swift Package Manager
+
+Flutter's Swift Package Manager support is turned off by default.
+To turn it on:
+
+1. Switch to Flutter's `main` channel:
+
+   ```sh
+   flutter channel main
+   ```
+
+2. Make sure you have the latest Flutter SDK:
+
+   ```sh
+   flutter upgrade
+   ```
+
+3. Turn on the Swift Package Manager feature:
+
+   ```sh
+   flutter config --enable-swift-package-manager
+   ```
+
+Using the Flutter CLI to run an app modifies the app to add Swift Package
```

Comment:
nit: does it modify the app, or modify the project?

Reply:
Technically both is true, but project is more precise. Updated!